snekbox/api/resources/eval.py

```diff
@@ -43,6 +44,7 @@ class EvalResource:
                     "required": ["path"],
                 },
             },
+            "binary_path": {"type": "string"},
```

Comment:
Maybe "executable_path" is a better name than "binary_path"? I'm also okay with the latter though.

Reply:
Yeah I agree, makes it clearer what is supported here, as it's not limitted to just binaries. https://github.com/python-discord/snekbox/pull/196/commits/edcaed6ad90f71ff61f6ceb7194de611ceeec592